scheduler/data-flow/src/main/kotlin/io/seldon/dataflow/Logging.kt

```diff
@@ -21,11 +21,12 @@ object Logging {
         appLevel: Level = Level.INFO,
         kafkaLevel: Level = Level.WARN,
     ) = loggingConfiguration {
-        kloggingMinLogLevel(appLevel)
```

Comment:
I think we can also keep kloggingMinLogLevel, set to appLevel, which sets the logging level for the logger itself: https://klogging.io/docs/internals/internal-logger/

Reply:
done in [5598a67](https://github.com/SeldonIO/seldon-core/pull/6350/commits/5598a67f4bf20d47b43b5ddcc90a1469a95faed2)